src/kensetsu-vaults.md

```diff
@@ -0,0 +1,106 @@
+# Kensetsu Platform
+
+## Theory
+
+### Overall description of the platform
+
+Kensetsu is a lending platform with an overcollateralised stablecoin pegged to the USD. It is a MakerDAO analogue in the SORA ecosystem.
```

Comment:
It worth to mention that USD is phase 1, the next phase adds more configurable stablecoins and peg to XAU (Gold) is planned.